src/main.py

```
@@ -227,7 +230,8 @@ def get_stats():
         return jsonify(stats)
 
     except Exception as e:
```

Pylint (4.0.4)

[warning] Catching too general exception Exception

(W0718)